lib/pluginmanager/util.rb

```diff
@@ -19,6 +19,11 @@ def self.logstash_plugin?(plugin, version = nil)
       dep = Gem::Dependency.new(plugin, version || Gem::Requirement.default)
       specs, error = Gem::SpecFetcher.fetcher.spec_for_dependency(dep)
 
+      # dump errors
+      error.each do |e|
+        $stderr.puts(e.wordy)
+      end
```

Comment:
Switch to plural form,`errors` instead of `error`?
Since it is a one liner we can use an inline block `{ |e|  .... }`